scalikejdbc-core/src/main/scala/scalikejdbc/TxBoundary.scala

```diff
@@ -41,10 +41,15 @@ object TxBoundary {
     }
   }
 
+  private[scalikejdbc] sealed abstract class TxBoundaryMissingImplicits {
+    implicit def `"To activate TxBoundary.Future, scala.concurrent.ExecutionContext value in implicit scope is required here."`[A]: TxBoundary[A] = sys.error("use internalTxBoundary!")
```

Comment:
I think this message should be understandable to library maintainers. 
How about `sys.error("This method will tell library users about missing implicit value by compilation error with the explanatory method name.")`?